core/tauri-runtime-wry/src/lib.rs

```diff
@@ -3342,27 +3400,7 @@ fn create_webview<T: UserEvent>(
       target_os = "ios",
       target_os = "android"
     ))]
-    WebviewKind::WindowChild => WebViewBuilder::new_as_child(&window),
-    WebviewKind::WindowContent => {
-      #[cfg(any(
-        target_os = "windows",
-        target_os = "macos",
-        target_os = "ios",
-        target_os = "android"
-      ))]
-      let builder = WebViewBuilder::new(&window);
-      #[cfg(not(any(
-        target_os = "windows",
-        target_os = "macos",
-        target_os = "ios",
-        target_os = "android"
-      )))]
-      let builder = {
-        let vbox = window.default_vbox().unwrap();
-        WebViewBuilder::new_gtk(vbox)
-      };
-      builder
-    }
+    WebViewBuilder::new_as_child(&window)
```

Comment:
iirc, using `new_as_child` has shortcomings on macOS and probably should not be used by default
cc @wusyong